website/content/en/preview/concepts/pod-density.md

```diff
@@ -24,6 +24,11 @@ Karpenter can be configured to disable nodes' ENI-based pod density.  This is es
 When using small instance types, it may be necessary to enable [prefix assignment mode](https://aws.amazon.com/blogs/containers/amazon-vpc-cni-increases-pods-per-node-limits/) in the AWS VPC CNI plugin to more pods per node.  Prefix assignment mode was introduced in AWS VPC CNI v1.9 and allows ENIs to manage a broader set of IP addresses.  Much higher pod densities are supported as a result.
 {{% /alert %}}
 
+{{% alert title="Windows Support Notice" color="warning" %}}
+Presently, only a single ENI per Windows worker node is supported.
+As a consequence, the number of IP addresses, and subsequently, the number of pods that a Windows worker node can support is limited by the number of IPv4 addresses available on the primary ENI.
```

Comment:
This is correct. However, at launch, Karpenter will only support secondary IP mode which means that the number of IPs available are equal to individual secondary IP addresses. With PD mode, the available IPs would be much higher but Karpenter would not support it.

We should add a sentence at the end of this para saying that at launch Karpenter will only support individual secondary IP addresses.

Reply:
Got it. Added a sentence being more explicit regarding your suggestion.